core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1186,6 +1186,14 @@ public VirtualHostBuilder defaultVirtualHost() {
         return defaultVirtualHostBuilder;
     }
 
+    /**
+     * Returns the {@link VirtualHostBuilder} for building the default
+     * <a href="https://en.wikipedia.org/wiki/Virtual_hosting#Name-based">name-based virtual host</a>.
+     */
+    public List<VirtualHostBuilder> defaultVirtualHostBuilders() {
```

Comment:
Could we hide this method from the public API. Our builder classes don't usually expose getter methods. Is it possible to make it package-private?

Reply:
Makes sense. Will do that.